docs/monitor-kubernetes.md

```diff
@@ -229,3 +263,36 @@ As a result, you have the `customresource-config-ksm` ConfigMap created.
 
       ![image](assets/images/pg_metric.svg)
 
+## Uninstall Victoria metrics Kubernetes stack
+
+To remove Victoria metrics Kubernetes stack used for Kubernetes cluster monitoring, use the cleanup script. By default, the script removes all the [Custom Resource Definitions(CRD)](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/) and Secrets associated with the Victoria metrics Kubernetes stack. To keep the CRDs, run the script with the `--keep-crd` flag.
+
+=== ":material-file-remove-outline: Remove CRDs"
+
+    Replace the `<NAMESPACE>` placeholder with the namespace you specified during the Victoria metrics Kubernetes stack installation: 
+
+    ```{.bash data-prompt="$" }
+    $ curl -fsL  https://raw.githubusercontent.com/Percona-Lab/k8s-monitoring/main/vm-operator-k8s-stack/cleanup.sh | bash -s -- --namespace <NAMESPACE>
```

Comment:
Made few changes in the uninstall script, command to run will also change now
```
bash <(curl -fsL https://raw.githubusercontent.com/Percona-Lab/k8s-monitoring/main/vm-operator-k8s-stack/cleanup.sh) --namespace <NAMESPACE>
```

Reply:
Updated